### DISCUSSION TITLE
Is it intended to be able to fork without an api key?

### DISCUSSION THREAD

**Discussion by CH1M4C** [ConsenSys-archive/ganache#3302] [Q&A]
compiled ganache embed infura api key, so provide access archive node without own api key.
[this](https://github.com/trufflesuite/ganache/pull/1579) branch name is "free-forking" so I'm confused about whether it is intended or not.
If it is not intended, I'll open a pull request.

**Comment by davidmurdoch** [comment]
We call it zero-config forking, and yes, it is a free feature.

Enjoy!
